package.json

```diff
@@ -35,14 +35,15 @@
     "react-dom": "^16.8"
   },
   "devDependencies": {
-    "@babel/cli": "^7.0.0",
-    "@babel/core": "^7.0.0",
-    "@babel/plugin-external-helpers": "^7.0.0",
+    "@babel/cli": "^7.6.2",
+    "@babel/core": "^7.6.2",
+    "@babel/plugin-external-helpers": "^7.2.0",
     "@babel/preset-flow": "^7.0.0",
     "@emotion/babel-preset-css-prop": "^10.0.14",
-    "@storybook/addon-info": "5.1.11",
-    "@storybook/addons": "5.1.11",
-    "@storybook/react": "5.1.11",
+    "@storybook/addon-docs": "^5.3.0-alpha.11",
+    "@storybook/addon-info": "^5.2.2",
```

Comment:
@ej9x I think you can delete this dependency.

Reply:
fixed